packages/react-ui-components/src/components/SchemaForm/Templates/DescriptionFieldTemplate/DescriptionFieldTemplate.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { DescriptionFieldProps } from '@rjsf/utils';
+import { KvInfoLabel } from '../../../stencil-generated';
+import styles from './DescriptionFieldTemplate.module.scss';
+import { isString } from 'lodash-es';
+
+const DescriptionFieldTemplate = ({ description }: DescriptionFieldProps) => {
+	if (isString(description)) {
+		return (
+			<div className={styles['description-container']}>
+				<KvInfoLabel labelTitle="" description={description} />
```

Comment:
If the label title is not needed in some cases maybe it should be required in the `kv-info-label` component.